src/Pages/GlobalConfigurations/DeploymentCharts/List/DeploymentChartsList.component.tsx

```diff
@@ -0,0 +1,173 @@
+/*
+ * Copyright (c) 2024. Devtron Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { useState } from 'react'
+import {
+    GenericEmptyState,
+    InfoColourBar,
+    APIResponseHandler,
+    useAsync,
+    Tooltip,
+    EMPTY_STATE_STATUS,
+    URLS,
+} from '@devtron-labs/devtron-fe-common-lib'
+import { DOCUMENTATION } from '@Config/constants'
+import emptyCustomChart from '@Images/ic-empty-custom-charts.png'
+import { ReactComponent as DevtronIcon } from '@Icons/ic-devtron-app.svg'
+import { ReactComponent as HelpIcon } from '@Icons/ic-help.svg'
+import { importComponentFromFELibrary } from '@Components/common'
+import UploadChartModal from './UploadChartModal'
+import { getChartList } from './service'
+import DeploymentChartsListHeader from './DeploymentChartsListHeader'
+import UploadButton from './UploadButton'
+import DownloadChartButton from './DownloadChartButton'
+import './styles.scss'
+
+const EditDeploymentChart = importComponentFromFELibrary('EditDeploymentChart', null, 'function')
+
+const DeploymentChartsList = () => {
+    const [showUploadPopup, setShowUploadPopup] = useState(false)
+    const [loading, list, error, reload] = useAsync(() => getChartList())
```

Comment:
Please avoid using name likes loading, list, error and reload since in case we add another API Call this will be highly misleading